docs/content/integration/openid-connect/flower/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Flower"
+description: "Integrating Flower with the Authelia OpenID Connect 1.0 Provider."
+summary: ""
+date: 2024-08-18T12:03:00+02:00
+draft: false
+images: []
+weight: 620
+toc: true
+support:
+  level: community
+  versions: true
+  integration: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Tested Versions
+
+* [Authelia]
+  * [v4.38.8](https://github.com/authelia/authelia/releases/tag/v4.38.8)
+* [Flower]
+  * [2.0.1 3b9ed2f](https://github.com/m4rc3l-h3/flower)
+
+{{% oidc-common %}}
+
+### Assumptions
+
+This example makes the following assumptions:
+
+* __Application Root URL:__ `https://flower.{{< sitevar name="domain" nojs="example.com" >}}/`
+* __Authelia Root URL:__ `https://{{< sitevar name="subdomain-authelia" nojs="auth" >}}.{{< sitevar name="domain" nojs="example.com" >}}/`
+* __Client ID:__ `flower`
+* __Client Secret:__ `insecure_secret`
+* __Authentication Name (Flower):__ `authelia`:
+    * This option determines the redirect URI in the format of
+      `https://flower.{{< sitevar name="domain" nojs="example.com" >}}/login`.
+      This means if you change this value you need to update the redirect URI.
```

Comment:
Not sure where the `authelia` element fits in here?

Reply:
Hi @james-d-elliott 
Thank you for your comments and sorry for the inconveniences. I have updated both aspects by a.) reworking the assumptions section regarding what is required from Flower and b.) added another section in the Application configuration to set the environment variable for flower.